packages/shared/src/sentry.ts

```diff
@@ -0,0 +1,126 @@
+import * as Sentry from "@sentry/node";
+const { nodeProfilingIntegration } = require("@sentry/profiling-node");
+import { AuthInfo } from "./graphql/fetchAuthInfoFromGraphQL";
+import { getDeviceId } from "./getDeviceId";
+import { randomUUID } from "crypto";
+import { formatTags, logger, Tags } from "./logger";
+
+const SENTRY_DSN =
+  "https://5c145b72bb502832982243d6584f163d@o437061.ingest.us.sentry.io/4507534763819008"; // write-only permissions
+
+class SentryAPI {
+  private deviceId: string;
+  private sessionId: string;
+  private authInfo?: AuthInfo;
+  private initialized: boolean = false;
+
+  constructor() {
+    this.deviceId = getDeviceId();
+    this.sessionId = randomUUID();
+  }
+
+  initialize(app: string, version: string | undefined) {
+    if (this.initialized) {
+      console.warn(`Logger already initialized.`);
+    }
+
+    this.initSentry(app, version);
+    this.initialized = true;
+  }
+
+  private initSentry(app: string, version: string | undefined) {
+    Sentry.init({
+      dsn: SENTRY_DSN,
+      integrations: [nodeProfilingIntegration()],
+    });
+
+    Sentry.setTags({
+      app,
+      version,
+    });
+  }
+
+  identify(authInfo: AuthInfo) {
+    this.authInfo = authInfo;
+  }
+
+  async close() {
+    if (process.env.REPLAY_TELEMETRY_DISABLED) {
+      return;
+    }
+
+    await Sentry.close();
+  }
+
+  async flush() {
+    if (process.env.REPLAY_TELEMETRY_DISABLED) {
+      return;
+    }
+
+    await Sentry.flush();
+  }
```

Comment:
Should we wrap/expose flush? Seems like we only want `Sentry.close` to be used so maybe it's clearer to only expose that.

Reply:
Yeah, we don't need this. I ended up not using it.